src/components/DataBrowser.vue

```diff
@@ -124,8 +136,25 @@ export default defineComponent({
             <patient-browser :patient-key="patient.key" />
           </v-expansion-panel-content>
         </v-expansion-panel>
+
+        <v-expansion-panel
+          v-if="dicomWeb.isConfigured"
+          :key="DICOM_WEB_KEY"
+          @change="handlePanelChange(DICOM_WEB_KEY)"
+        >
+          <v-expansion-panel-header>
+            <v-icon class="collection-header-icon">mdi-cloud-download</v-icon>
+            <span
+              >{{ `${dicomWeb.hostName || dicomWeb.host} | DICOMWeb` }}
```

Comment:
The auto-formatter sometimes screws this one up.

```suggestion
            <span>
              {{ `${dicomWeb.hostName || dicomWeb.host} | DICOMWeb` }}
```

Reply:
Fixed